server/config.js

```diff
@@ -40,12 +40,14 @@ module.exports = {
   api: {
     hubEndpoint,
     hubContent: `${hubEndpoint}/v1/api/content`,
+    hubContentRouter: `${hubEndpoint}/router/translate-path`,
```

Comment:
would not worry about adding to this - these paths aren't configuration so shouldn't really be here and can go into the client. 

So would just use `hubEndpoint` and hardcode `/router/translate-path` in the client